WeatherApp/WeatherAppTests/Controllers/WeatherForecastControllerTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NSubstitute;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WeatherApp.WebSite.Controllers;
+using WeatherApp.WebSite.Models;
+using WeatherApp.WebSite.Services;
+
+namespace WeatherApp.Tests.Controllers
+{
+    [TestFixture]
+    class WeatherForecastControllerTest
+    {
+        private IWeatherForecastService _weatherForecastService;
+        private WeatherForecastController _weatherForecastController;
+
```

Comment:
Avoid unnecessary blank lines, it only stretches the overall code.

Reply:
Thanks for the remark, it's fixed now.